src/app/pages/component-viewer/component-viewer.scss

```diff
@@ -1,3 +1,9 @@
+@import '../../../main';
```

Comment:
Main.scss imports quite a lot from material and adding an import of main here will duplicate what it brings in which is quite substantial. I believe you can safely remove this line anyways since you are not using any of the variables or mixins it declares